include/wx/math.h

```diff
@@ -52,13 +52,13 @@
 
 
 #ifdef __cplusplus
+#include <float.h>
```

Comment:
Since we have problem under MSW only I think  something like this would be OK here:
#if defined(__WINDOWS__)
    #include <cfloat>
#endif

This would fix issues with all C++ compilers so another includes wouldn't be necessary.